tests/stacks/examples/alice/test_data/expected/admissionregistration.k8s.io_v1beta1_mutatingwebhookconfiguration_inferenceservice.serving.kubeflow.org.yaml

```diff
@@ -41,6 +41,10 @@ webhooks:
     matchExpressions:
     - key: control-plane
       operator: DoesNotExist
+  objectSelector:
```

Comment:
just note that this moves the minimal kube requirement to 1.15, not exactly sure what's the min requirement for kubeflow now.

Reply:
Yea, not quite sure either as the compatibility matrix here https://www.kubeflow.org/docs/started/k8s/overview/ doesn't seem to have been updated for 1.1.